llama_hub/firebase_realtimedb/base.py

```diff
@@ -0,0 +1,74 @@
+"""Firebase Realtime Database Loader."""
+
+from typing import Dict, List, Optional
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+import firebase_admin
```

Comment:
make lazy import in `__init__` and `load_data` (see other loaders for examples)

Reply:
Sure @jerryjliu, thank you for your feedback! I have made the required changes and implemented lazy import in both the __init__ and load_data functions. Please take a look at the updated code. Your further review would be greatly appreciated. Thanks again!